Header header

uint32 redArea
uint32 redX
uint32 redY
uint32 redLeft
uint32 redRight
uint32 redTop
uint32 redBottom

uint32 greenArea
uint32 greenX
uint32 greenY
uint32 greenLeft
uint32 greenRight
uint32 greenTop
uint32 greenBottom

uint32 blueArea
uint32 blueX
uint32 blueY
uint32 blueLeft
uint32 blueRight
uint32 blueTop
uint32 blueBottom
